sonar-plugin-api/src/main/java/org/sonar/api/security/ExternalGroupsProvider.java

```diff
@@ -19,20 +19,54 @@
  */
 package org.sonar.api.security;
```

Comment:
Please ignore the changes in these files. These changes are done as part of the SONAR-6856 ( We have created separate pull request for this)

SONAR-6856: Add a new API ExternalGroupsProvider.doGetGroups(Context context) to allow plugins to pass groups information to SonarQube